src/org/ods/OdsContext.groovy

```diff
@@ -385,12 +385,7 @@ class OdsContext implements Context {
       returnStdout: true, script: 'git config --get remote.origin.url',
       label : 'getting GIT url'
     ).trim()
-    // check if the url already contains a user
-    if (!gitUrl.contains('@')) {
-      return gitUrl.replace('://', '://cd_user@')
-    } else {
-      return gitUrl
-    }
+    return gitUrl
```

Comment:
Are you 100% sure? I am worried that we did this replacement for a reason 😆

Reply:
I would suggest, we keep all project specifix cd user PRs open until end of week or thursday evening.

Reply:
OK. I tried to trace usages of this variable and I would agree that theoretically it should be fine. The image label will change (in `stageStartOpenshiftBuild`), but that should be OK.